src/digestif_conv.ml

```
@@ -77,7 +77,7 @@ struct
   let consistent_of_hex_opt hex =
     match consistent_of_hex hex with
     | digest -> Some digest
-    | exception _ -> None
+    | exception Invalid_argument _ -> None
```

there are other places in the code with catch all handler.

---

Where ? I did not see them.